docs/apply.md

```diff
@@ -0,0 +1,61 @@
+# How to apply
+
+1.  Read this reference guide, which will help you to apply and perform the different tests needed to evaluate your
+    solution.
+2.  If it is needed, set up the validation environment.
+3.  Execute tests according to the test workflow using the appropriate public instances or using your own deployed
+    instance.
+4.  Fill in the
+    [**FIWARE-ready** IoT Device Manual Template](https://s3.amazonaws.com/fundingbox-sites/gear%2F1517994518659-FIWARE+Ready+IoT+Device+Manual+Template+v1+%282%29.docx)
+    with the information about how to use your component within FIWARE ecosystem.
```

Comment:
FIWARE Ecosystem? What do you mean here?

Reply:
We use **Ecosystem** to mean the software (since we don't want to use **platform** since we are not an IoT Platform.) This complements with our consistent terminology **community** meaning the people and institutions creating that software.